Gemfile

```diff
@@ -11,5 +11,5 @@ if Dir.exist?(logstash_path) && use_logstash_source
 end
 
 if RUBY_VERSION == "1.9.3"
-  gem 'rake', '12.2.1'
-end
\ No newline at end of file
+  gem 'rake', '12.3.3'
```

Comment:
This change cannot be made as rake `12.3.0` and above requires ruby version > 2.0, which Logstash `6.x` and above requires.

We might want to think about just removing this block entirely (and in other plugins) - this is here to retain `5.x` compatibility, but we haven't tested this plugin (or many others) against `5.x` for some time.

Thoughts @jsvd ?